job-service-dropwizard/pom.xml

```
@@ -102,6 +102,16 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-text</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-server</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId> org.eclipse.jetty.toolchain</groupId>
```

There is an extra space here.

```suggestion
                    <groupId>org.eclipse.jetty.toolchain</groupId>
```